src/prefect/tasks/gsheets/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""
+A collection of tasks for interacting with Google Sheets.
+"""
+try:
+    from prefect.tasks.gsheets.gsheets import (
+        GsheetUpdates,
+        GsheetResponse,
+        WriteGsheetRow,
+        ReadGsheetRow,
+        gsheet_helper,
+    )
+except ImportError:
+    raise ImportError(
+        'Using `prefect.tasks.gsheets` requires Prefect to be installed with the "gsheets" extra.'

```

Comment:
You named the extra `"gspread"` so we should update this error message accordingly

Reply:
Fixed!  Changed the extra's name to `gsheets`

Reply:
Not sure what to do with the changelog, though?  I don't see a `changes/` directory.